src/UI/Root/Root.tsx

```diff
@@ -1,18 +1,17 @@
 import React, { useContext, useEffect, useMemo, useState } from "react";
 import { Navigate, Route, Routes } from "react-router-dom";
-import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
 import { ReactQueryDevtools } from "@tanstack/react-query-devtools";
 import { LoginPage } from "@/Slices/Login";
 import { DependencyContext } from "@/UI/Dependency";
 import { SearchSanitizer } from "@/UI/Routing";
 import { GlobalStyles } from "@/UI/Styles";
 import { NotFoundPage } from "@S/NotFound/UI";
-import { KeycloakProvider, PageFrame, Initializer } from "./Components";
+import RouteOutlet from "../Routing/RouteOutlet/RouteOutlet";
```

Comment:
Why have a separate folder for RouteOutlet? 
And if you really want a separate folder, it should use an index file.